util/keeper/address_test.go

```diff
@@ -0,0 +1,20 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec/address"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/palomachain/paloma/testutil"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestValAddressConversion(t *testing.T) {
```

Comment:
- Verify return value matches expected value
- Add a test case for invalid address

Reply:
added these two cases with assertion